File: sdk/list_services/list_services/DiscoveryInformation.m
/*******************************************************************************
 * Copyright (c) Microsoft Open Technologies, Inc.
 * All Rights Reserved
 * Licensed under the Apache License, Version 2.0.
 * See License.txt in the project root for license information.
 ******************************************************************************/

#import "DiscoveryInformation.h"

@implementation DiscoveryInformation

- (BaseEntity *)createFromJson:(NSDictionary *)data{
    self.Capability = [data valueForKey:@"Capability"];
    self.ServiceName = [data valueForKey:@"ServiceName"];
    self.ServiceEndpointUri = [data valueForKey:@"ServiceEndpointUri"];
    self.ServiceResourceId = [data valueForKey:@"ServiceResourceId"];
    //dinformation.Id = diccionary.valueForKey("id") as String;
    
    return self;
}

@end

File: Sdk/office365-base-sdk/office365-base-sdk/DiscoveryInformation.swift
//
//  DiscoveryInformation.swift
//  office365-base-sdk
//
//  Created by Lagash on 6/13/14.
//  Copyright (c) 2014 microsoft. All rights reserved.
//

import Foundation

class DiscoveryInformation: BaseEntity{
    var ServiceName : String = "";
    var ServiceResourceId: String = "";
    var Capability : String = "";
    var ServiceEndpointUri: String = "";
}